### ISSUE TITLE
Doesn't work with xcode 10 as a cocoa pod

### ISSUE THREAD

**Issue by ntkon** [luximetr/AnyFormatKit#10]
When I build an existing project (that was fine in xcode 9) with xcode 10, I get the following error:

```
Multiple commands produce '/Users/xxxUserNamexxx/Library/Developer/Xcode/DerivedData/xxxAppxxx-budrinxtnjnzsobhgwboeiqsfcsk/Build/Products/Debug-iphonesimulator/AnyFormatKit/AnyFormatKit.framework/Info.plist':
1) Target 'AnyFormatKit' (project 'Pods') has copy command from '/Users/xxxAppLocationxxx/Pods/AnyFormatKit/Source/Info.plist' to '/Users/xxxUserNamexxx/Library/Developer/Xcode/DerivedData/xxxAppxxx-budrinxtnjnzsobhgwboeiqsfcsk/Build/Products/Debug-iphonesimulator/AnyFormatKit/AnyFormatKit.framework/Info.plist'
2) Target 'AnyFormatKit' (project 'Pods') has process command with output '/Users/xxxUserNamexxx/Library/Developer/Xcode/DerivedData/xxxAppxxx-budrinxtnjnzsobhgwboeiqsfcsk/Build/Products/Debug-iphonesimulator/AnyFormatKit/AnyFormatKit.framework/Info.plist'
```
I removed it from the podfile and used carthage and then it works. Looks like there is something wrong in the pod specifications?

**Comment by luximetr** [comment]
Ok, thank about it. Also you can remove plist file in pods project settings from compiler sources.
21 сент. 2018 г., 02:04 +0300, ntkon <notifications@github.com>, писал:

**Comment by devanshvyas** [comment]
This issue is solved, please accept pull request https://github.com/luximetr/AnyFormatKit/pull/17

**Comment by jvanzummeren** [comment]
Has fixed release in version 0.1.3
